.changes/next-release/feature-DynamoDBEnhancedClient-a84a047.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "DynamoDBEnhancedClient",
+    "description": "Support of ReturnConsumedCapacity in DynamoDBEnhancedClient's BatchWrite and TransctWriteItems operations",
```

Comment:
Can we have separate PRs for support `BatchWrite` and `TransctWriteItems`? 
Maybe we can start with `batchWrite` first since it's more straightforward?

Reply:
Sure, just reverted my changes for `TransactWriteItems`

Reply:
@zoewangg How would you like to approach this PR? https://github.com/aws/aws-sdk-java-v2/pull/5501/files
Would you like to review this? Since you've already provided some feedback

Reply:
Thanks! We'll follow our SOP to find a reviewer for #5501 :) cc @debora-ito

Reply:
@zoewangg Let me know if you have any feedback for BatchWriteItem